sw/device/tests/power_virus_systemtest.c

```diff
@@ -684,6 +684,49 @@ static void configure_pinmux_sim(void) {
   //    Channel 0 on IOR11
   CHECK_DIF_OK(dif_pinmux_output_select(&pinmux, kTopEarlgreyPinmuxMioOutIor11,
                                         kTopEarlgreyPinmuxOutselPwmAonPwm0));
+
+  if (kDeviceType == kDeviceSilicon || kDeviceType == kDeviceSimDV) {
```

Comment:
We can probably do the attribute configuration unconditionally once #24688 is in.